src/auth/auth-config.ts

```diff
@@ -816,6 +906,8 @@ export class OIDCConfig implements OIDCAuthProviderConfig {
       providerId: this.providerId,
       issuer: this.issuer,
       clientId: this.clientId,
+      clientSecret: this.clientSecret,
```

Comment:
What if clientSecret is undefined?

Reply:
Fixed by setting clientSecret to optional.